website_and_docs/content/documentation/getting_started/how_to_upgrade_to_selenium_4.ko.md

```diff
@@ -408,10 +396,8 @@ driver.manage().timeouts().pageLoadTimeout(Duration.ofSeconds(10));
 {{< /card >}}
 {{< /cardpane >}}
 
-Waits are also expecting different parameters now. `WebDriverWait` is now expecting a `Duration` 
-instead of a `long` for timeout in seconds and milliseconds. The `withTimeout` and `pollingEvery` 
-utility methods from `FluentWait` have switched from expecting `(long time, TimeUnit unit)` to 
-expect `(Duration duration)`.
+
+대기 중에도 다른 매개 변수가 필요합니다. `WebDriverWait`는 이제 몇 초, 몇 밀리 초 단위로 시간이 초과되는 'long' 대신 'Duration'을 기대하고 있습니다. 'FluentWait'의 'withTimeout'과 'pollingEvery' 유틸리티 방식은 '(long time, TimeUnit unit)'로 예상되던 것이 '(Duration duration)'을 예상하는 것으로 변경되었습니다.
 
```

Comment:
Here, instead of a single quote for words, you can use backtick so it will look like below in docs
With single quote => 'withTimeout'과 'pollingEvery'
with backtick => `withTimeout`과 `pollingEvery`